### PR TITLE
Clarified upgrade requirements for Forms 10

### PR DESCRIPTION
**Pull request by AndyButland**
This adds a couple of points around upgrades that came up in community discussions.

### CODE REVIEW

File: Add-ons/UmbracoForms/Installation/Version-Specific.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ This page covers specific upgrade documentation for specific versions.
 
 Version 10 of Umbraco Forms has a minimum dependency on Umbraco CMS core of `10.0.0`. It runs on .NET 6.
 
-To migrate to version 10 you should first update to the latest minor release of version 9. This will ensure you have all the database schema changes in place.
+To migrate to version 10 you should first update to the latest minor release of version 9. If you are upgrading from Umbraco 8, you can also update Forms to the latest minor version of Forms 8, and ensure you have the configuration in place for [storing form definitions in the database](../Developer/Forms-in-the-Database/index-v8.md).
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Installation/Version-Specific.md:14]:
```suggestion
To migrate to version 10, you should first update to the latest minor release of version 9. If you are upgrading from Umbraco 8, update Forms to the latest minor version of Forms 8 and ensure you have the configuration in place for storing form definitions in the database. For more information, see the [Umbraco Forms in the Database](../Developer/Forms-in-the-Database/index-v8.md) article.
```

File: Add-ons/UmbracoForms/Installation/Version-Specific.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ This page covers specific upgrade documentation for specific versions.
...
+
+Either will ensure you have all the database schema changes in place.
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Installation/Version-Specific.md:16]:
```suggestion
Either way, ensure you have all the database schema changes in place.
```

**Review by eshanrnh — APPROVED** [review]
Looks good, @AndyButland. 💪Just a small suggestion to include the article/topic title as the hyperlink title.

File: Add-ons/UmbracoForms/Installation/Version-Specific.md (hunk 2 above)
Comment (AndyButland) [reply Add-ons/UmbracoForms/Installation/Version-Specific.md:16]: This should probably be "Either way will... " - I'll make that change.

**Comment by eshanrnh** [comment]
Thanks @AndyButland 👏 Merging it in.

**Pull request merged**